websocket-ballerina/tests/sync_connection_close.bal

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 WSO2 Inc. (//www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// //www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/io;
+import ballerina/lang.runtime as runtime;
+
+string closeError = "";
+listener Listener l32 = new(21002);
+service /onCloseText on l32 {
+   resource function get .() returns Service|UpgradeError {
+       return new WsServiceSyncClose();
+   }
+}
+
+service class WsServiceSyncClose {
+  *Service;
+  remote isolated function onTextMessage(Caller caller, string data) returns Error? {
```

Comment:
```suggestion
    remote isolated function onTextMessage(Caller caller, string data) returns Error? {
```
Can you recheck the indentation of code? Tab space is generally 4 characters in our development.